core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -374,43 +373,55 @@ def validate_funding_spent(workbook: dict[str, pd.DataFrame]) -> list["TownsFund
             ]
     else:
         # check funding against individual project funding allocated for Towns Deal submissions
-        return [
-            TownsFundRoundFourValidationFailure(
-                sheet="Funding",
-                section=f"Project Funding Profiles - Project {get_project_number_by_id(project_id, project_ids)}",
-                column="Grand Total",
-                message=msgs.OVERSPEND_PROJECT,
-                # first project begins after 17 and each project is seperated by 28 cells
-                row_index=17 + 28 * get_project_number_by_id(project_id, project_ids),
-            )
-            for project_id in project_ids
-            if round(funding_spent[project_id], 2) > DefaultConfig.TF_FUNDING_ALLOCATED[project_id]
-        ]
-
-    return funding_spent_failures
+        funding_spent_failures = []
+        for expense_type in ["CDEL", "RDEL"]:
+            for idx in project_ids:
+                project_number = get_project_number_by_id(idx, project_ids)
+                if round(funding_spent[expense_type][idx]) > get_allocated_funding(idx, expense_type):
+                    funding_spent_failures.append(
+                        TownsFundRoundFourValidationFailure(
+                            sheet="Funding",
+                            section=f"Project Funding Profiles - Project {project_number}",
+                            column="Grand Total",
+                            message=msgs.OVERSPEND.format(expense_type=expense_type),
+                            row_index=13 + 28 * project_number if expense_type == "CDEL" else 16 + 28 * project_number,
+                        )
+                    )
+        if funding_spent_failures:
+            return funding_spent_failures
```

Comment:
I think we can just return`funding_spent_failures` regardless of if its truthy or not